lib/Core/Search/Solr/DocumentMapper.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Netgen\EzPlatformSearchExtra\Core\Search\Solr;
+
+use eZ\Publish\SPI\Persistence\Content;
+use EzSystems\EzPlatformSolrSearchEngine\DocumentMapper as DocumentMapperInterface;
+use Netgen\EzPlatformSearchExtra\Core\Search\Solr\SubdocumentMapper\ContentSubdocumentMapper;
+use Netgen\EzPlatformSearchExtra\Core\Search\Solr\SubdocumentMapper\ContentTranslationSubdocumentMapper;
+
+/**
+ * Indexes information on whether Content field value is empty.
```

Comment:
This does not seem right? :)

Reply:
Thanks, fixed in https://github.com/netgen/ezplatform-search-extra/pull/3/commits/9c17623cf13bf9f05ffa9855f2e12721f435af29.